06a-WGS.Rmd

```diff
@@ -0,0 +1,118 @@
+- [Strelka](https://github.com/Illumina/strelka) This is a small variant caller designed by illumina. It is used for identifying germline variants in cohorts of samples and somatic variants in tumor/normal sample pairs. 
+- [SomaticSniper](https://gmt.genome.wustl.edu/packages/somatic-sniper/) SomaticSniper can be used to identify SNPs in tumor/normal pairs. It calculates the probability that the tumor and normal genotypes are different and reports this probability as a somatic score. 
+- [Pindel](https://www.sanger.ac.uk/tool/pindel/) Pindel is a tool that uses a pattern growth approach to detect breakpoints of large deletions, medium size insertsion/inversion, tandem duplications. 
+
+Here are commonly used variant calling annotation tools: 
+- [Annovar](https://annovar.openbioinformatics.org/en/latest/) This is a helpful tool for annotating, filtering, and combining the output data from the above tools. It can be used for gene-based, region-based, or filter-based annotations. 
+- [GENCODE](https://www.gencodegenes.org/#) This tool can be used to identify and classify gene features in human and mouse genomes. 
+- [dbSNP](https://www.ncbi.nlm.nih.gov/snp/) This is a resource to look up specific human single nucleotide variations, microsatellites, and small-scale insertions and deletions. 
+- [Ensembl](https://uswest.ensembl.org/index.html) This resource is a genome browser for annotating genes from a wide variety of species. 
+
+These are some of the commonly used copy number variation analysis tools: 
+- [GATK](https://gatk.broadinstitute.org/hc/en-us/articles/360035531092--How-to-part-I-Sensitively-detect-copy-ratio-alterations-and-allelic-segments) GATK has a variety of tools that ca nbe used to study changes in copy numbers of genes. This link provides a tutorial for how to use the tools. 
+- [AscatNGS](https://www.crick.ac.uk/research/labs/peter-van-loo/software) This tools (allele-specific copy number analysis of tumors) is specific for WGS copy number variation analysis. It can be used to dissect allele-specific copy numbers of tumors by estimating and adjusting for tumor ploidy and nonaberrant cell admixture.  
+- [TitanCNA](https://github.com/gavinha/TitanCNA) This tool, developed by Fred Hutch Principal Investigator Dr. Gavin Ha, is used to analyze copy number variation and loss of heterozygoisity at the subclonal level for both WGS and WES data in tumors compared to matched normals. This tool accounts for mixtures of cell populations and estimates the proportion of cells harboring each event. The Ha lab has developed a [snakemake pipeline](https://github.com/gavinha/TitanCNA/tree/master/scripts/snakemake) to more easily use this tool.   
```

Comment:
```suggestion
- [TitanCNA](https://github.com/gavinha/TitanCNA) This tool, developed by Fred Hutch Principal Investigator Dr. Gavin Ha, is used to analyze copy number variation and loss of heterozygoisity at the subclonal level for both WGS and WES data in tumors compared to matched normals. This tool accounts for mixtures of cell populations and estimates the proportion of cells harboring each event. The Ha lab has developed a [snakemake pipeline](https://github.com/gavinha/TitanCNA/tree/master/scripts/snakemake) to more easily use this tool.   
```

This is the only time you've mentioned who developed a tool. And while I understand why you are mentioning that we may need to be a little more subtle about it. Can we link to the paper and put that at the end of the paragraph? Something like `Ha et al published this about TitanCNA`